### DISCUSSION TITLE
Windows 11 Start menu with Windows 10 system tray icons

### DISCUSSION THREAD

**Discussion by kchurch05** [valinet/ExplorerPatcher#1483] [General]
Apologies if this has been answered, but I wasn't able to find anything in the help or wiki.

I like the look of the Windows 11 start menu, but hate the grouped flyout on the system tray icons (namely wifi). However, when I set my system tray to Windows 10, my start menu also has the Windows 10 skin - even though the start menu style is set as Windows 11.

Is this meant to happen? TIA!

**Comment by vippium** [comment]
try to restart the explorer...if this won't fix it. then, Reinstall the EP

**Comment by pyrates999** [comment]
Please show a screen shot of what you are seeing.

**Comment by kchurch05** [comment]
I've restarted several times, but keep seeing the same result. First image - Taskbar set to Window 10, expected behavior. Second one - Start Menu set as Windows 11 but showing Windows 10 format.

![image](https://github.com/valinet/ExplorerPatcher/assets/17455352/48f681ee-dc2a-40e8-ad4d-933f37c2cb1f)

![image](https://github.com/valinet/ExplorerPatcher/assets/17455352/b64707b1-0579-4836-a6c1-6abc41b15b07)

**Reply by vippium** [reply]
I think that you are using something else with EP (like StartIsBack or OpenShell). As I can see the Start.
